app/javascript/src/components/GameLogEntry.vue

```diff
@@ -1,17 +1,132 @@
 <template>
-  <div class="border border-black rounded p-2 m-2">
+  <div v-if="Array.isArray(event)" class="border border-black rounded p-2 m-2 rondel">
+    <div v-for="(action, index) in event" :key="(index + Math.random()).toString()">
```

Comment:
I'd like to get `Math.random()` out of here if possible. Don't worry about the duplicate index error with `GB0`, I've resolved that elsewhere in `Province` (and it's entirely unrelated to your changes here). Can we make the key just be `index`?

Reply:
Yes, it looks like we can change it back to only `index`.